new-integration-tests/src/test/resources/wdt-models/wdt-singleclusterdomain-sampleapp-wls.yaml

```diff
@@ -0,0 +1,32 @@
+# Copyright (c) 2020, Oracle Corporation and/or its affiliates.
+# Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+
+domainInfo:
+    AdminUserName: '@@PROP:AdminUserName@@'
```

Comment:
Can you replace @@PROP with 
AdminUserName: '@@SECRET:__weblogic-credentials__:username@@
AdminPassword: '@@SECRET:__weblogic-credentials__:password@@'

In the latest WDT version we support @@SECRET.

Reply:
That is not working when I generate domain in image. Can we use this firstly and in the meanwhile let me communicate with WIT team.

Reply:
It is fine for the first check-in